examples/formik/js/async-validator/helper.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  font-family: ;
```

Comment:
This is empty

Reply:
You are right, but going to skip updates on this CSS, is from formik examples